app/src/main/res/layout/activity_sign_up.xml

```diff
@@ -0,0 +1,196 @@
+        android:layout_alignBottom="@+id/ivProfilePhoto"
+        android:layout_alignParentEnd="true"
+        android:layout_marginStart="15dp"
+        android:layout_marginEnd="30dp"
+        android:layout_marginBottom="4dp"
+        android:layout_toEndOf="@+id/ivProfilePhoto"
+        android:background="@drawable/edit_text_background"
+        android:hint="@string/username"
+        android:inputType="textPersonName"
+        android:padding="5dp"
+        android:paddingStart="8dp"
+        android:textColor="@color/white"
+        android:textColorHint="#6BFFFFFF"
+        android:textSize="14sp"
+        android:textStyle="normal" />
+
+    <EditText
+        android:id="@+id/etFirstName"
+        android:layout_width="170dp"
+        android:layout_height="40dp"
+        android:layout_below="@+id/ivProfilePhoto"
+        android:layout_alignParentStart="true"
+        android:layout_marginStart="30dp"
+        android:layout_marginTop="25dp"
+        android:background="@drawable/edit_text_background"
+        android:hint="@string/first_name"
+        android:inputType="textPersonName"
+        android:padding="5dp"
+        android:paddingStart="8dp"
+        android:textColor="@color/white"
+        android:textColorHint="#6BFFFFFF"
+        android:textSize="14sp"
+        android:textStyle="normal" />
+
+    <EditText
+        android:id="@+id/etLastName"
+        android:layout_width="170dp"
+        android:layout_height="40dp"
+        android:layout_alignBottom="@+id/etFirstName"
+        android:layout_alignParentEnd="true"
+        android:layout_marginStart="10dp"
+        android:layout_marginEnd="30dp"
+        android:layout_marginBottom="0dp"
+        android:layout_toEndOf="@+id/etFirstName"
+        android:background="@drawable/edit_text_background"
+        android:hint="@string/last_name"
+        android:inputType="textPersonName"
+        android:padding="5dp"
+        android:paddingStart="8dp"
+        android:textColor="@color/white"
+        android:textColorHint="#6BFFFFFF"
+        android:textSize="14sp"
+        android:textStyle="normal" />
+
+    <EditText
+        android:id="@+id/etEmail"
+        android:layout_width="144dp"
+        android:layout_height="40dp"
+        android:layout_below="@+id/etFirstName"
+        android:layout_alignParentStart="true"
+        android:layout_alignParentEnd="true"
+        android:layout_marginStart="31dp"
+        android:layout_marginTop="20dp"
+        android:layout_marginEnd="29dp"
+        android:background="@drawable/edit_text_background"
+        android:hint="@string/email"
+        android:inputType="textWebEmailAddress"
+        android:padding="5dp"
+        android:paddingStart="8dp"
+        android:textColor="@color/white"
+        android:textColorHint="#6BFFFFFF"
+        android:textSize="14sp"
+        android:textStyle="normal" />
+
+    <EditText
+        android:id="@+id/etPersonalDescription"
+        android:layout_width="161dp"
```

Comment:
should the width here be 144 to match the other edit texts? Might be worth making a dimension resource for some of these reused sizes but that's up to you!

Reply:
Not, this size is auto because is constraint to the image view and the parent end